vaadin-popover-flow-parent/vaadin-popover-flow/src/main/java/com/vaadin/flow/component/popover/Popover.java

```diff
@@ -60,6 +68,48 @@ public class Popover extends Component implements HasComponents {
      */
     public Popover() {
         getElement().getNode().addAttachListener(this::attachComponentRenderer);
+
+        setTrigger(Set.of(PopoverTrigger.CLICK));
+    }
+
+    /**
+     * Sets the popover trigger mode describing how the overlay is opened or
+     * closed on user interaction with the target.
+     *
+     * When passing null or an empty set, the popover can only be opened and
+     * closed programmatically.
```

Comment:
This is to be added in a follow-up PR?

Reply:
There is already #6337 that adds API to set `opened` state.